### PR TITLE
Create folder for WG content 

### PR DESCRIPTION
**Pull request by vikas-agarwal76**

### CODE REVIEW

**Comment by vikas-agarwal76** [comment]
Creating a content folder to store content (documents, compliance artifacts JSONs, etc.) created by Compliance WG.

**Review by mnm678 — APPROVED** [review]

File: community/working-groups/compliance/content/readme.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This folder contains content created by Compliance WG
```
Comment (brandtkeller) [inline community/working-groups/compliance/content/readme.md:1]: I'll take a look at the current warnings we are seeing the linting process. Nonetheless the linting action is failing as this file needs a top level H1.

**Review by brandtkeller — CHANGES_REQUESTED** [review]

File: community/working-groups/compliance/content/readme.md (hunk 1 above)
Comment (brandtkeller) [reply community/working-groups/compliance/content/readme.md:3]: Appreciated

**Review by brandtkeller — COMMENTED** [review]
Branch just needs to be rebased to bring it up-to-date.

**Comment by vikas-agarwal76** [comment]
> Branch just needs to be rebased to bring it up-to-date.

done

**Comment by vikas-agarwal76** [comment]
@brandtkeller I rebased the branch as suggested by you. Kindly merge.

**Review by brandtkeller — APPROVED** [review]

**Comment by brandtkeller** [comment]
No issues with the current state. 

Provided this is a placeholder for others to contribute and iterate in a central location - than looks good to me. 

I would recommend adding context in your PR descriptions towards why the change should be made. This placeholder will be published to the website and I would usually recommend adding this directory when you have changes to be proposed that include substance.

**Pull request merged**